pipeline/installs/multi-user/pipelines-profile-controller/sync.py

```diff
@@ -0,0 +1,274 @@
+# Copyright 2020 Google LLC
```

Comment:
What is this doing?

Reply:
@Bobgy it looks like this is a controller to setup a namespace for pipelines? Instead of using a controller why  not just publish a kustomize / kpt package that makes it easy to spin up a namespace with all the resources a user needs?
This is in line with
https://github.com/kubeflow/manifests/issues/1136

Reply:
Your understanding is correct. #1136 does seem a better approach to me!

I'd prefer merging this as is, so that we can start testing KFP multi user mode.
Then we can remove it when #1136 is done. I'd be happy to contribute a package for setting up KFP namespace.

Reply:
What do you think?

Reply:
Why is this blocked on #1136?  It looks like your controller is just creating a bunch of K8s resources in the namespace in which KFP is going to be used.

Could you just create a kustomize package containing those resources and then tell users that to setup a namespace to
use KFP they need to do something like

```
<kustomize it for their namespace; e.g. using kpt or >
kustomize build  | kubectl apply -f -
```

Reply:
I think current profile controller provides self service, everyone coming can just use right away. It makes more sense if the deployment is done automatically for them.

I understand customization needs exist for sure, #1136 will likely be the best way forward. I need some time to think about it. Efforts lacking is setting up the package and some documentation to use them. I'd want to avoid doing too much things in one step.

My personal priority is getting this PR and a second PR with istio authorization in, so that previously designed KFP multi user mode is working properly first. Then I can start thinking about substantial improvements like the approach in #1136 .